index.js

```diff
@@ -1,34 +1,40 @@
-'use strict';
+'use strict'
+
+const defaultPackages = [
+  'apollo-cache',
+  'apollo-cache-inmemory',
+  'apollo-client',
+  'apollo-link',
+  'apollo-link-context',
+  'apollo-link-http',
+  'graphql',
+  'graphql-tools',
+  'graphql-tag',
+];
 
 module.exports = {
   name: 'ember-apollo-client',
 
-  included() {
+  included(app) {
     this._super.included.apply(this, arguments);
-
+    
+    let config = app.project.config(app.env) || {};
+    this.addonConfig = config['apollo'] || {};
```

Comment:
This is more sugar

```js
this.addonConfig = this.app.project.config(app.env)['apollo'] || {};
```

Reply:
Is there a case where `this.app.project.config(app.env)` could be `undefined`? I don't feel strongly about this though, the existing way is a bit easier to skim.

Reply:
No! This is the better way, `ember-cli-mirage` use like this.